src/verso.rs

```diff
@@ -75,10 +77,33 @@ impl Verso {
     /// - Canvas: Enabled
     /// - Constellation: Enabled
     /// - Image Cache: Enabled
-    pub fn new(evl: &ActiveEventLoop, proxy: EventLoopProxy<()>, config: Config) -> Self {
+    pub fn new(
+        evl: &ActiveEventLoop,
+        proxy: EventLoopProxy<EventLoopProxyMessage>,
+        config: Config,
+    ) -> Self {
+        if let Some(ipc_channel_name) = &config.args.ipc_channel_name {
+            let sender =
+                IpcSender::<IpcSender<ControllerMessage>>::connect(ipc_channel_name.to_string())
+                    .unwrap();
+            let (controller_sender, receiver) = ipc::channel::<ControllerMessage>().unwrap();
+            sender.send(controller_sender).unwrap();
+            let proxy_clone = proxy.clone();
+            std::thread::spawn(move || {
+                while let Ok(message) = receiver.recv() {
+                    if let Err(e) = proxy_clone
+                        .send_event(EventLoopProxyMessage::WebviewControllerMessage(message))
+                    {
+                        log::error!("Failed to send controller message to Verso: {e}");
+                    }
+                }
+            });
```

Comment:
I don't think this needs to be another thread. Could it be another method to call after(or before) handling winit event?

Reply:
I think we need to wake up the event loop? I think other ones like the embedder will send a wake message to the event loop from its own thread when receiving a new message and then we check the message in `handle_servo_messages`, this is similar to that

Reply:
I see. Let's use thread builder to give it a name. And could you rename `WebviewControllerMessage`(and other similar ones) to IPCMessage? We will document what does this mean in details.

Reply:
I think it can also be optional if there's no IPC channel.

Reply:
Not sure what do you mean, this part will only run if there's an `ipc-channel` arg specified from cli

Reply:
oh my bad. You already did it.